From 59daf398bc0f1d7895eee3a776b33a9c9310ad21 Mon Sep 17 00:00:00 2001
From: Alexander Tsoy <alexander@tsoy.me>
Date: Sun, 26 Oct 2014 23:54:47 +0300
Subject: configure.ac: workaround quoting issues

BROWSER_PLUGIN_DIR variable substitution is not portable. In particular
it does not work in dash. Replace it with conditional.

https://bugzilla.gnome.org/show_bug.cgi?id=739226

diff --git a/configure.ac b/configure.ac
index 56bf93c..dd4bcab 100644
--- a/configure.ac
+++ b/configure.ac
@@ -455,7 +455,9 @@ AC_ARG_ENABLE([browser-plugin],
 if test x$enable_browser_plugin = "xyes" ; then
   PKG_CHECK_MODULES([BROWSER_PLUGIN],[gtk+-3.0 >= $GTK_REQUIRED gthread-2.0 gio-2.0 >= $GLIB_REQUIRED])
 
-  BROWSER_PLUGIN_DIR="${BROWSER_PLUGIN_DIR:-"\${libdir}/mozilla/plugins"}"
+  if test -z "${BROWSER_PLUGIN_DIR}"; then
+    BROWSER_PLUGIN_DIR="\${libdir}/mozilla/plugins"
+  fi
   AC_ARG_VAR([BROWSER_PLUGIN_DIR],[Where to install the plugin to])
 fi
 
-- 
cgit v0.10.1

